changelog.d/5806.bugfix

```diff
@@ -0,0 +1 @@
+Fix get_user_deactivated_status being inaccessible to synapse/handlers/auth.py. This was causing a 500 error every time a deactivated user attempted to login.
```

Comment:
Change log entries should ideally say what the end users or server admins will see, rather than what happened in the code.

```suggestion
Fix error when trying to login as a deactivated user when using a worker to handle login.
```